orgs-using-d.dd

```diff
@@ -222,6 +222,15 @@ $(DIVC orgs-using-d center,
             $(FA_HIRING github.com/symmetryinvestments/overview/blob/master/hiring.md)
         )
     )
+    $(DORG Tagion, https://www.tagion.org/, tagion.org,
+        Decentralized DLT network,
```

Comment:
You might want to clarify that DLT means (I'm assuming) Distributed ledger technology here. Not everyone is going to know what that means. 

(Also the linked site does not say what DLT is/means, it assumes you know).

Reply:
Thanks... On it ;-)

Reply:
Is this better @thewilsonator?